pkg/logs/input/journald/docker.go

```diff
@@ -42,5 +45,12 @@ func (t *Tailer) getContainerTags(containerID string) []string {
 
 // initializeTagger initializes the tag collector.
 func (t *Tailer) initializeTagger() {
+	var taggerInstance tagger.Tagger
```

Comment:
the logs component always run in the `core-agent` so here local agent is enough.